PYTHIA8/AliPythia8/AliPythia8.h

```diff
@@ -104,6 +105,7 @@ class AliPythia8 :public AliTPythia8, public AliPythiaBase
     Float_t               fPtScale;           //  ! cut-off joining scale
     Int_t                 fNJetMin;           //  ! min. number of jets
     Int_t                 fNJetMax;           //  ! max. number of jets
+    Bool_t				  fDecayLonglived;	  //    Decay long-lived particles
```

Comment:
Formatting is done. Btw. do you think we can import the .clang-format file from O2, in case someone likes to use it?